include/shellstrings.h

```diff
@@ -19,7 +31,7 @@
  * Returns:
  *  - None.
  */
-void shell_prompt();
+void shell_prompt(bool *color);
 
 /*
  * shell_intro - Introduction text for the shell.
```

Comment:
Please update this header to accurately reflect the changed function.

Reply:
updated!